contracts/PartyBid.sol

```diff
@@ -391,7 +390,7 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
      * @return _maxBid the maximum bid
      */
     function _getMaximumBid() internal view returns (uint256 _maxBid) {
```

Comment:
if we're in here already it'd be nice to make this public but not necessary

Reply:
I was mainly trying not to change the ABI of `PartyBid` for this round of changes